apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -83,15 +91,65 @@ const AliasTokensSwatchList: React.FunctionComponent = () => {
     return Object.keys(aliasColorTokens).map(aggregator);
   }, [aliasColorTokens, aggregator]);
 
-  const aliasTokensAsArray = React.useMemo(flattenArray, [flattenArray]);
-  const renderSwatch = React.useCallback((item) => {
-    const { color, key } = item;
-    return <ColorToken key={key} color={color} name={key} />;
-  }, []);
+  const aliasColorTokensAsArray = React.useMemo(flattenArray, [flattenArray]);
+
+  const renderSwatch = React.useCallback(getSwatch, []);
+
+  return (
+    <View style={commonTestStyles.view}>
+      <View style={styles.stackStyle}>{aliasColorTokensAsArray.map((item) => renderSwatch(item))}</View>
+    </View>
+  );
+};
+
+const globalNeutralColorNamesSortedDarkToLight = Object.keys(globalTokens.color)
+  .filter((globalColorName) => globalColorName.includes('grey') || globalColorName === 'black' || globalColorName === 'white')
+  .sort((color1, color2) => {
+    const color1hex = globalTokens.color[color1];
```

Comment:
Nit: colors aren't guaranteed to be a hex so maybe this should be rethought? Not a blocking comment.

Reply:
Oh I assumed they were. What else can the color value be?

Reply:
If this is still a concern feel free to reopen

Reply:
They can be a platformColor, or a Dynamic Color. Something that looks like:
`semantic: { 'windowBackgroundColor' }}` or `{light: 'hex', dark: 'hex'}

Reply:
Every platform can define their own types, so it's not worth assuming the structure.